conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -681,14 +569,8 @@ public void setSeekEnd() {
         @Override
         public void onEvent(PlayerEvent.AdStarted adStartedEvent) {
             Log.d(TAG, "[Player Event] AdStarted");
-            Client.AdPosition adPosition = AdEventUtil.parseAdPosition(adStartedEvent, bitmovinPlayer.getDuration());
             adStarted = true;
-
-            try {
-                client.adStart(sessionId, Client.AdStream.SEPARATE, Client.AdPlayer.CONTENT, adPosition);
-            } catch (ConvivaException e) {
-                Log.e(TAG, e.getLocalizedMessage());
-            }
+            convivaVideoAnalytics.reportAdBreakStarted(ConvivaSdkConstants.AdPlayer.CONTENT, ConvivaSdkConstants.AdType.SERVER_SIDE);
```

Comment:
Same comment as AdTrackEnd.
Also, should this use  ConvivaSdkConstants.AdType.CLIENT_SIDE instead? This AdStarted event is fired only for client side Ads.

Reply:
Yes. Changed that.

Reply:
thanks. Will resolve this when the related code change is committed.